pom.xml

```
@@ -1,51 +1,147 @@
+    <packaging>pom</packaging>
+    <url>http://projects.spring.io/spring-xd/</url>
+    <version>1.0.0.BUILD-SNAPSHOT</version>
+    <organization>
+        <name>Pivotal Software, Inc.</name>
+        <url>http://www.spring.io</url>
+    </organization>
+    <properties>
+        <spring-cloud.version>Brixton.BUILD-SNAPSHOT</spring-cloud.version>
+        <spring-framework.version>4.2.0.RC2</spring-framework.version>
+        <spring-integration.version>4.2.0.M2</spring-integration.version>
+    </properties>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.3.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+
+    <dependencyManagement>
+
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream-codec</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream-common</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream-binder-local</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream-binder-redis</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
```

@mbogoevici Actually, with the current autoconf, one will be silently ignored, which is no better